src/images/shaun-shaker.webp

```diff
@@ -0,0 +1 @@
+
```

Comment:
Please can you replace this image with this one:

![shaun-shaker](https://github.com/shaunsaker/shaun-saker-v2/assets/19655207/37660a59-657a-45e6-b3da-bb3d15dd2e23)

You'll need to also update the import in src/pages/projects/data.tsx to use new jpg format instead of webp.